### DISCUSSION TITLE
Maximum number of clusters in ArgoCD?

### DISCUSSION THREAD

**Discussion by ryjogo** [argoproj/argo-cd#18469] [Q&A]
Hi, What is the maximum, supported number of clusters and applications in ArgoCD?

**Comment by agaudreault** [comment]
There are no maximum, but depending on the quantity of clusters and the resources in them, you will need to configure different argocd parameters to scale the instance.

**Reply by ryjogo** [reply]
Interesting, I have a use case for about 3.5k and growing edge clusters. Not so many deployments in them though. What parameters should I start to take a look at that could make this work ?

**Reply by agaudreault** [reply]
https://argo-cd.readthedocs.io/en/stable/operator-manual/high_availability/#scaling-up is a good place to start
